omnibus/files/private-chef-ctl-commands/backup.rb

```diff
@@ -25,10 +25,15 @@
     end
   end.parse!(ARGV)
 
-  Chef::Mixin::DeepMerge.deep_merge!(stringify_keys(options.to_h), running_config['private_chef']['backup'])
+  begin
+    Chef::Mixin::DeepMerge.deep_merge!(stringify_keys(options.to_h), running_config['private_chef']['backup'])
+    ChefBackup::Runner.new(running_config).backup
+  rescue => e
+    puts "[ERROR] #{e.message}"
+    exit(1)
+  end
 
-  status = ChefBackup::Runner.new(running_config).backup
-  exit(status ? 0 : 1)
+  exit(0)
```

Comment:
the way omnibus-ctl works you can just 'return 0' , and above 'return 1'. That will be used as the exit code.